packages/contracts-bedrock/test/libraries/Hashing.t.sol

```diff
@@ -136,3 +136,39 @@ contract Hashing_hashDepositTransaction_Test is CommonTest {
         );
     }
 }
+
+contract Hashing_hashSuperRootProof_Test is CommonTest {
+    /// @notice Tests that the Solidity impl of hashSuperRootProof matches the FFI impl
+    /// @param _timestamp The timestamp of the super root proof
+    /// @param _length The number of output roots in the super root proof
+    /// @param _seed The seed used to generate the output roots
+    function testDiff_hashSuperRootProof_succeeds(uint64 _timestamp, uint256 _length, uint256 _seed) external {
+        // Ensure at least 1 element and cap at a reasonable maximum to avoid gas issues
+        _length = uint256(bound(_length, 1, 50));
```

Comment:
If 50 is a conservative upper bound you can leave this, but you can add the `noGasMetering` modifier to this test if you were getting out of gas issues

Reply:
We can make it higher than 50, I'll do some testing to see what's viable.